app/workers/display_ad_event_rollup_worker.rb

```diff
@@ -0,0 +1,10 @@
+class DisplayAdEventRollupWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :low_priority
+
+  def perform
+    month_ago = Date.current - 32.days
+    DisplayAdEventRollup.rollup month_ago
```

Comment:
Just to verify, this is designed to only go through events of a day that was a month ago?

Reply:
:+1: The PR that added tracking for logged-out users (https://github.com/forem/forem/pull/18460) was merged 22 days ago. (I don't imagine this rollup service will have significant effects on rows for logged-in users -- unless some user is clicking on *lots* of ads?)

So, we store a high amount of granularity in tracking for about a month -- moment by moment, how active were these ads -- then we compress these events to a daily record.